docs/source/utils/notebook.py

```diff
@@ -13,7 +13,12 @@ def inner(example_text: str):
 \"\"\"
 
 # %%
-!python3 -m pip install -q dff[examples]
+!python3 -m pip install -q dff[examples] # Installs dff with dependencies for running examples
+# To install the minimal version of dff, use `pip install dff`
+# To install other options of dff, use `pip install dff[OPTION_NAME]`
```

Comment:
One can also install multiple dependencies:
`pip install dff[ydb, mysql]`
This should be mentioned.